### PR TITLE
Support for coffeescript 2 keywords and cjsx 

### PR DESCRIPTION
**Pull request by aminland**

### CODE REVIEW

**Comment by GeoffreyBooth** [comment]
Thank you for doing this! How can I test this out?

**Comment by aminland** [comment]
@GeoffreyBooth you can test by uninstalling first, then adding the repository for my fork via `package control: add repository` before reinstalling

File: CoffeeScript.tmLanguage [hunk 1]
```diff
@@ -222,12 +222,32 @@
...
+					<string>keyword.control.coffee</string>
+				</dict>
+				<key>2</key>
+				<dict>
+					<key>name</key>
+					<string>keyword.control.coffee</string>
+				</dict>
+			</dict>
+			<key>match</key>
+			<string>\b(?:(for)(?:\b.+\b)(from))\b</string>
```
Comment (GeoffreyBooth) [inline CoffeeScript.tmLanguage]: This doesn’t appear to catch all uses. In particular I’m looking in the CoffeeScript repo:

https://github.com/jashkenas/coffeescript/blob/c4245e50c2fda100c258110727ac8e76d2ac1a92/test/arrays.coffee#L252-L289

I don’t see any of these `from` keywords highlighted (though I do see `await` highlighted elsewhere, so I should have your fork loaded).

**Comment by GeoffreyBooth** [comment]
This doesn’t appear to catch all uses. In particular I’m looking in the CoffeeScript repo:

https://github.com/jashkenas/coffeescript/blob/c4245e50c2fda100c258110727ac8e76d2ac1a92/test/arrays.coffee#L252-L289

I don’t see any of these from keywords highlighted (though I do see await highlighted elsewhere, so I should have your fork loaded).

**Comment by GeoffreyBooth** [comment]
Also:

image

**Pull request closed without merging**